src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleExpanded.java

```diff
@@ -64,24 +70,28 @@ enum E4 {
 
 @interface Another4 {
     String[] value() default {};
-    @Another({"foo", "bar"}) //compact style // violation
+    // violation below 'Annotation style must be 'EXPANDED''
+    @Another({"foo", "bar"}) //compact style
     String value1() default "";
 }
 
 @SomeArrays(pooches = {})
-@Another({}) // violation
+// violation below 'Annotation style must be 'EXPANDED''
+@Another({})
```

Comment:
Please use comment above ONLY if there is no space to fix this comment on same line.
Sorry that I was not clear on this from the beginning.

Please apply this to all other cases like this

Reply:
Done. Changes have been made accordingly.